docs/concepts/style_guide.md

```diff
@@ -0,0 +1,178 @@
+# Beaker Style Guide
+
+## Scope of this guide
+
+The purpose of this guide is to provide definitions for preferred practices when
+writing Beaker tests, both syntactically and stylistically. This guide will
+define and provide examples for preferred test layout and conventions. Common
+patterns that are recommend as well as patterns that should be avoided will be
+described.
+
+No style manual can cover every possible circumstance. When a judgement call
+becomes necessary, keep in mind the following general ideas:
+
+1. **Readability matters**. If you have to choose between two equally effective
+alternatives, pick the more readable one. This is, of course, subjective, but if
+you can read your own code three months from now, that’s a great start. Don’t be
```

Comment:
nit pick, replace smart quote in `that's` and `don't` with simple ascii single quote